beast/physicsmodel/prior_weights_stars.py

```diff
@@ -57,8 +61,9 @@ def compute_age_prior_weights(logages, age_prior_model):
             np.array(age_prior_model["values"]),
         )
     elif age_prior_model["name"] == "exp":
-        vals = (10 ** logages) / (age_prior_model["tau"] * 1e6)
-        age_weights = np.exp(-1.0 * vals)
+        # assumes SFR(t) \propto e**(-t/tau) where age=(t0-t) and tau in Gyr
+        vals = (10 ** logages) / (age_prior_model["tau"] * 1e9)
```

Comment:
The comment implies that the time is shifted by t0, but then the code in the next line does not do this.  Is this a mistake in the comment or the code?

Reply:
I've clarified this better now via new commit where comment now reads:
```
        # assumes SFR(t) \propto e**(-t/tau) \propto e**(age/tau)
        # where age \propto -t (for age=t0-t) and tau in Gyr
```